Hardware/Hardware/CardReader/CardReaderConfigProvider.swift

```diff
@@ -1,6 +1,16 @@
 /// Abstraction provided by Hardware so that clients of this library
 /// can model a way to provide a connection token.
 /// It is meant to abstract an implementation of the [adapter pattern](https://en.wikipedia.org/wiki/Adapter_pattern)
-public protocol CardReaderConfigProvider {
+
+public protocol ReaderTokenProvider {
     func fetchToken(completion: @escaping(String?, Error?) -> Void)
 }
+
+public protocol ReaderLocationProvider {
+    func fetchDefaultLocationID(completion: @escaping(String?, Error?) -> Void)
+}
+
+public protocol CardReaderConfigProvider: ReaderLocationProvider {
```

Comment:
```suggestion
public protocol CardReaderConfigProvider: ReaderLocationProvider, ReaderTokenProvider {
```

🔧 It looks like this protocol also includes `ReaderTokenProvider`, but there's no explicit conformance to that one, I think we should add it.

Reply:
Opened https://github.com/woocommerce/woocommerce-ios/issues/5424